crates/sui-framework/sources/governance/staking_pool.move

```diff
@@ -415,6 +421,12 @@ module sui::staking_pool {
 
     public fun delegation_token_amount(delegation: &Delegation): u64 { balance::value(&delegation.pool_tokens) }
 
+    public fun pool_token_exchange_rate(pool: &StakingPool): PoolTokenExchangeRate {
```

Comment:
so to get the latest exchange rate, the "client" wants to make a move call of this function?

Reply:
It is also available as an event emitted by `emit_validator_epoch_events`, which is called by `advance_epoch`

Reply:
> to get the latest exchange rate

The exchange rate is updated once per epoch during epoch change so the rate in the event is the latest